UI/Pages/Cookies.cshtml

```diff
@@ -114,6 +119,11 @@
                     <td class="govuk-table__cell">Saves your cookie consent settings</td>
                     <td class="govuk-table__cell">1 year</td>
                 </tr>
+                   <tr>
+                    <td class="govuk-table__cell">.FindATuitionPartner.Mvc.CookieTempDataProvider</td>
+                    <td class="govuk-table__cell">Stores your search parameters</td>
+                    <td class="govuk-table__cell">1 year</td>
```

Comment:
This one is a session cookie I believe so it should be "When the browser is closed"

Reply:
Changed with suggestion added